ssl/ssl_test.cc

```diff
@@ -12116,16 +12119,13 @@ TEST_P(BadKemKeyShareFinishTest, BadKemKeyShareFinish) {
   {
     // The server's public key was already correctly generated previously in
     // a call to Accept(). Here we modify it.
```

Comment:
It would be good to modify this comment too to say the public key is randomly generated as in the test above.

Reply:
done